tests/vppinputasync.h

```diff
@@ -28,18 +28,20 @@ class VppInputAsync : public VppInput
 
     bool read(SharedPtr<VideoFrame>& frame);
 
+    /*startupSize means how many buffers will preload to memory in startup, it will overide queueSize*/
     static SharedPtr<VppInput>
-    create(const SharedPtr<VppInput>& input, uint32_t queueSize);
+    create(const SharedPtr<VppInput>& input, uint32_t queueSize, uint32_t startupSize = 0);
```

Comment:
No, this is intent to, we can't change m_startupSize at running time. It needs reallocate resource and hold some lock.